app/seeders/demo_data/work_package_seeder.rb

```diff
@@ -207,12 +207,13 @@ def create_relation(to:, from:, type:)
     end
 
     def calculate_start_date(days_ahead)
-      monday = Date.today.monday
-      days_ahead > 0 ? monday + days_ahead : monday
+      Time.zone.today.monday + (days_ahead || 0).days
     end
 
+    # Returns the due date based on the starting date and the duration
+    # but ensures that the due date cannot be before the start date.
     def calculate_due_date(date, duration)
-      duration && duration > 1 ? date + duration : date
+      [date + ((duration || 0) - 1).days, date].max
```

Comment:
This one here looks much better than the older line. However, I believe that they produce different results as we now subtract one day from the duration. Is that a problem?

Reply:
The manually groomed seed data has been adapted and looks fine AFAIK. But for BIM, the seed pulling might need to be adapted.